go/cmd/vttablet/vttablet.go

```diff
@@ -74,7 +74,8 @@ func main() {
 	}
 
 	// creates and registers the query service
-	qsc := tabletserver.NewServer()
+	ts := topo.Open()
```

Comment:
This function is creating a new object, it should actually be called 'New'. We used to have a single 'global' topo server, and that was a bad pattern that made a whole bunch of tests really hard to manage.